src/iniReader/readinifile.cpp

```diff
@@ -0,0 +1,68 @@
+// ****** readinifile.cpp **********
+// ****  William R. Good  ********
+
+#include "XPLMCamera.h"
+#include "XPLMDataAccess.h"
+#include "XPLMDisplay.h"
+#include "XPLMGraphics.h"
+#include "XPLMMenus.h"
+#include "XPLMPlanes.h"
+#include "XPLMPlugin.h"
+#include "XPLMProcessing.h"
+#include "XPLMUtilities.h"
+
+#include "inireader.h"
+#include "FlyWithLua.h"
+#include <vector>
+
+#include <fstream>
+#include <iostream>
+//#include <string>
+#include <cstring>
+
+
+// ***** Configuration File Process ******
+void flywithlua::process_read_ini_file()
+
+{
+  char *iniDefaultPluginPath;
+  int found_file = 0;
+
+  iniDefaultPluginPath =
+      const_cast<char *>("./Resources/plugins/FlyWithLua/fwl_prefs.ini");
+//  string version;
+
+  cleanupIniReader();
+
+  std::ifstream ifile2(iniDefaultPluginPath);
+  if (ifile2) {
+    if (found_file == 0) {
+      XPLMDebugString("\nFlyWithLua: Found fwl_prefs.ini in the FlyWithLua plugin "
+                        "path and it is\n");
+      XPLMDebugString(iniDefaultPluginPath);
+      XPLMDebugString("\n");
+
+      parseIniFile(iniDefaultPluginPath);
+//      xbtn2cmdini_path_name = iniDefaultPluginPath;
+    }
+
+  } else {
+    return;
+  }
+
+  // Do we want to enable developer mode?
+  readOptionAsInt("DeveloperMode", &developer_mode);
+
+//  }
+
+//  version = getOptionToString("Version");
+//  XPLMDebugString("\nXbtn2cmd: xbtn2cmd.ini is version   ");
+//  XPLMDebugString(version.c_str());
+//  XPLMDebugString("\n");
+
+  // ****************** page one   ***************************
+
+//  page1_button_label = getOptionToString("page1_button_label");
```

Comment:
Leftover debugging code? I think we can safely remove it.

Reply:
Done